### PR TITLE
LINK-1894 | env variable to enable Swedish translations

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Enable Swedish translation if REACT_APP_ENABLE_SWEDISH_TRANSLATIONS is set to true. Also remove TODO-comments related to Swedish version and add tests to Swedish content.

Env variable should be added to Platta as well. Open PR to do that: https://dev.azure.com/City-of-Helsinki/linkedevents/_git/linkedevents-pipelines/pullrequest/8048

## Closes
[LINK-1894](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1894)

[LINK-1894]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1894?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr348-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr348-ui.dev.hel.ninja :laughing::tada::tada::tada:

File: src/domain/app/header/__tests__/Header.test.tsx [hunk 1]
```diff
@@ -206,19 +212,43 @@ test('should show mobile menu', async () => {
...
-  const enOption = getElement('enOption');
-  await user.click(enOption);
+  for (const item of testCases) {
+    await user.click(item[0]);
+    expect(history.location.pathname).toBe(item[1]);
+  }
+});
 
-  expect(history.location.pathname).toBe('/en');
+test('should not display Swedish language is Swedish translations are not enabled', async () => {
```
Comment (harriris-vincit) [inline src/domain/app/header/__tests__/Header.test.tsx:239]:
```suggestion
test('should not display Swedish language if Swedish translations are not enabled', async () => {
```

**Review by harriris-vincit — APPROVED** [review]
One tiny improvement suggestion, but overall LGTM and worked well when testing 👍

File: src/domain/app/header/__tests__/Header.test.tsx (hunk 1 above)
Comment (jorilindell) [reply src/domain/app/header/__tests__/Header.test.tsx]: Fixed the typo

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr348-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr348-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**